doc/manual/src/command-ref/nix-store/realise.md

```diff
@@ -8,33 +8,36 @@
 
 # Description
 
-The operation `--realise` essentially “builds” the specified store
-paths. Realisation is a somewhat overloaded term:
-
-  - If the store path is a *derivation*, realisation ensures that the
-    output paths of the derivation are [valid] (i.e.,
-    the output path and its closure exist in the file system). This
-    can be done in several ways. First, it is possible that the
-    outputs are already valid, in which case we are done
-    immediately. Otherwise, there may be [substitutes]
-    that produce the outputs (e.g., by downloading them). Finally, the
-    outputs can be produced by running the build task described
-    by the derivation.
-
-  - If the store path is not a derivation, realisation ensures that the
-    specified path is valid (i.e., it and its closure exist in the file
-    system). If the path is already valid, we are done immediately.
-    Otherwise, the path and any missing paths in its closure may be
-    produced through substitutes. If there are no (successful)
-    substitutes, realisation fails.
 
+Each of *paths* is processed as follows:
```

Comment:
Help out a reader who hasn't read the definition of realisation yet?

```suggestion
The goal of realisation is to make outputs exist.

Each of *paths* is processed as follows:
```

Reply:
Alternatively, on line 3

```diff
-`nix-store --realise` - realise specified store paths
+`nix-store --realise` - make outputs exist
```

Reply:
What are outputs? What does it mean for them to exist? I'm all for providing a motivation/introduction, but it should convey something meaningful to readers who don't already know all the answers.

We originally had

```suggestion
Ensure that the given store paths are [valid].

Each of *paths* is processed as follows:
```

but that was dismissed in review because it doesn't explain anything, and we decided to go into the algorithm directly.

Reply:
``nix-store --realise` - realise specified store paths` is rather useless though. It's a cyclic definition, and all it tells me is that store paths are involved, and I think I would expect 80% of the `nix-store` command to be about its contents anyway: store paths.

Surely we can think of something that conveys more than a fractional bit of information?

    `nix-store --realise` - substitute and/or build store paths

Reply:
I still think that anchoring the language we use to store paths is misleading. What about

> build or fetch store objects

Reply:
Sgtm. Store path is just muscle memory, and we do use it as the key to address store objects.